src/main/java/com/rocketden/main/controller/v1/GameController.java

```diff
@@ -0,0 +1,40 @@
+package com.rocketden.main.controller.v1;
+
+import com.rocketden.main.dto.room.GetRoomRequest;
+import com.rocketden.main.dto.room.RoomDto;
+import com.rocketden.main.service.RoomService;
+import com.rocketden.main.dto.game.StartGameRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GameController extends BaseRestController {
+    
+    private final RoomService service;
+
+    @Autowired
+    public GameController(RoomService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/start")
+    public ResponseEntity<Void> startGame(@RequestBody StartGameRequest request) {
```

Comment:
I'm not really sure it's necessary, since in both cases someone who is not the host is attempting to use host rights. A normal non-host user wouldn't "accidentally" stumble upon the start game button. Do you think it makes a difference? If so I'm open to adding it.

Reply:
That's fair reasoning, ok no worries, you don't have to add this (we can reopen the discussion if @alankbi thinks different, but resolving now).